### ISSUE TITLE
Support did:keri

### ISSUE THREAD

**Issue by andorsk** [TBD54566975/ssi-sdk#140]
https://identity.foundation/keri/did_methods/#:~:text=A%20close%20analogy%20is%20did,wise%20and%20n%2Dwise%20purposes.

**Comment by decentralgabe** [comment]
I'd like to hear a user / use case for this one before we implement it. It seems incredibly complex, and I'm not certain of the utility.

**Comment by andorsk** [comment]
Personally, I'm mostly interested in off-ledger did methods. KERI has a number of advantages over did:peer. I'd want to implement this personally. 

https://github.com/decentralized-identity/kerigo

There's a number of things keri does that help, but the main thing is it solves for making SSI portable. But to your point earlier: yes. It is complex. I have personally had a tough time wrapping my head around it. 🙄 I'm not sure the current kerigo is where it needs to be for production 🤷 but tbh I need to learn more. 

Totally understand the reservations here, and it's also brushing against the Vision issue 
https://github.com/TBD54566975/ssi-sdk/issues/111.

**Comment by decentralgabe** [comment]
My main concern is implementing a feature no one uses, and it going stale. This is not to say features won't need updating, they certainly will. But perhaps its a good initial bar to set for new features.

If you are stating that you will in fact be using KERI, then it does seem reasonable to include it in the library.

**Comment by andorsk** [comment]
Yep. I intend to use (or at least try to use ) `did:keri`. I'm not sure who else will need it though. I'll need to play more with the current tools though to understand what's available already on DIF
